style.css

```diff
@@ -737,16 +807,558 @@ ul {
+  }
+
+  .card-container {
+    display: flex;
+    float: left;
+    flex-direction: row;
+    padding: 24px;
+    gap: 46px;
+    border: 1px solid #dfe1e6;
+    margin-top: 100px;
+    height: 60vh;
+    width: 100%;
+    border-radius: 24px;
+  }
+
+  .card-container img {
+    width: 50%;
+    height: 100%;
+    border-radius: 8px;
+  }
+
+  .faha2 {
+    order: 2;
+  }
+
+  .work-fonony {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    padding-left: 2%;
+    margin-left: 10%;
+    width: 50%;
+    margin-top: 5rem;
+  }
+
+  .card-tittle {
+    margin-top: -15rem;
+    margin-left: -5.5rem;
+  }
+
+  .frame2-img {
+    margin-top: -1rem;
+    margin-left: -5.7rem;
+  }
+
+  .frame3-img {
+    display: flex;
+    align-items: center;
+  }
+
+  .project-info {
+    list-style: none;
+    display: flex;
+    justify-content: space-evenly;
+    margin: 0;
+    width: 60%;
+    font-size: 16px;
+    margin-left: -8.5rem;
+    margin-top: -3rem;
+    margin-bottom: 1rem;
+    font-weight: bold;
+    flex-direction: row;
+    gap: 8px;
+    order: 2;
+  }
+
+  ul.project-info li {
+    list-style-type: none;
+    position: relative;
+    white-space: nowrap;
+  }
+
+  ul.project-info li:not(:first-child)::before {
+    content: "\2022";
+    font-size: 40px;
+    position: absolute;
+  }
+
+  .fb {
+    font-size: 18px;
+    font-weight: 500;
+    line-height: 24px;
+    text-align: left;
+    color: #3b5998;
+  }
+
+  .full {
+    font-size: 18px;
+    font-weight: 400;
+    line-height: 24px;
+    text-align: left;
+    color: #6b778c;
+  }
+
+  .year {
+    font-size: 18px;
+    font-weight: 400;
+    line-height: 24px;
+    text-align: left;
+    color: #6b778c;
+  }
+
+  #fb-title {
+    margin-top: -15rem;
+    margin-left: -5.5rem;
+  }
+
+  #fb-info {
+    list-style: none;
+    display: flex;
+    justify-content: space-evenly;
+    margin: 0;
+    width: 60%;
+    font-size: 16px;
+    margin-left: -8.5rem;
+    margin-top: -3rem;
+    margin-bottom: 1rem;
+    font-weight: bold;
+    flex-direction: row;
+    gap: 8px;
+    order: 2;
+  }
+
+  #back-list {
+    margin-left: -5.3rem;
+    margin-top: -10.5rem;
+  }
+
+  #fd-buttons {
+    margin-left: -5.4rem;
+    margin-top: 5rem;
+  }
+
+  #description-fb {
+    margin-top: -2rem;
+    width: 66%;
+    margin-left: -5.3rem;
+  }
+
+  .description {
+    display: flex;
+    margin-top: -2rem;
+    width: 66%;
+    margin-left: -5.3rem;
+  }
+
+  ul.languages-list li {
+    white-space: nowrap;
+  }
+
+  .languages-list {
+    margin-left: -5.3rem;
+    margin-top: -8.5rem;
+  }
+
+  .buttons {
+    margin-left: -5.4rem;
+    margin-top: 2rem;
+    background: #4053fc;
+    color: #fff;
+  }
+
+  /* About section */
+
+  .about-me {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-content: center;
+    width: 100%;
+    height: 60vh;
+    padding: 56px 14px 0 16px;
+    margin-left: 3rem;
+  }
```

Comment:
- [ ] Nice effort making the changes :+1: But your "About Myself" section is totally disarranged. Kindly check it.

### Your design 
![Screenshot from 2023-04-08 19-08-58](https://user-images.githubusercontent.com/91142432/230739099-989bf4cb-9724-477e-b2c8-dd47a091f4a1.png)

### Figma design 
![Screenshot from 2023-04-05 06-38-27](https://user-images.githubusercontent.com/91142432/230739118-bc6024b2-dbb6-46eb-badb-8bc42de58b32.png)